definitions/divorce/json/AuthorisationCaseType/AuthorisationCaseType.json

```diff
@@ -57,7 +57,7 @@
     "LiveFrom": "16/05/2019",
     "CaseTypeID": "DIVORCE_NOTICE_OF_ACTING",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
```

Comment:
This should remain "CRU" I think.
I think what you were looking for is the **event** called "Notice of Acting". The ID for this is actually "create". 
(This is for element 8 in the ticket right?)
If yes, then you have nothing to change there or on the "create" event it would need to stay "CRU". This is the event that respondent solicitors use to link themselves to a case.
But we have to put "CRU" for both solicitors as the [RESPSOLICITOR] case role is not yet present before that "create" event is triggered.
Hope that helps

Reply:
I had a lot of issues with this area today. However, thanks to your message, it really helped me make sense out of it; and potentially i have found an issue with out tests. 
```const jsonConfigFiles = mergeJsonFilesFor('definitions/divorce/json/AuthorisationCaseType/', 'AuthorisationCaseType-');``` I think this needs to be changed to ```const jsonConfigFiles = mergeJsonFilesFor('definitions/divorce/json/AuthorisationCaseType/', '-nonprod');```

Reply:
This is the correction you asked for - https://github.com/hmcts/div-ccd-definitions/pull/634/commits/aeea6bf938d867526b11eb09087fbff29bf530c2